src/logic.ts

```diff
@@ -16,9 +16,9 @@ $(function () {
 const separators: string[] = ['-', '.', '_'];
-const nouns: string[] = ['order', 'butter', 'ticket', 'smoke', 'birds', 'rock', 'wing', 'fog', 'sky', 'water', 'cream', 'bikes', 'shoes', 'vegetable', 'cow', 'fork', 'haircut', 'waves', 'dinosaurs', 'blood', 'servant', 'structure', 'party', 'clown', 'pen', 'throat', 'frogs', 'earthquake', 'card', 'toothpaste', 'spy', 'worm', 'activity', 'truck', 'potato', 'office', 'island', 'bottle', 'fairies', 'maid', 'needle', 'squirrel', 'cherry', 'stone', 'advertisement', 'bells', 'kittens', 'deer', 'hobbies', 'river', 'bubble', 'soup', 'cemetery', 'coal', 'crime', 'sugar', 'thunder', 'grandfather', 'ants', 'steam', 'aftermath', 'hair', 'respect', 'wound', 'ground', 'door', 'gold', 'wish', 'fire', 'ring', 'dogs', 'theory', 'watch', 'machine', 'soda', 'star', 'locket', 'name', 'berry', 'castle', 'reward', 'pollution', 'talk', 'school', 'reaction', 'celery', 'canvas', 'car', 'airplane', 'pet', 'cakes', 'grass', 'breath', 'hand'];
+const nouns: string[] = ['order', 'butter', 'ticket', 'smoke', 'birds', 'rock', 'wing', 'fog', 'sky', 'water', 'cream', 'bikes', 'shoes', 'vegetable', 'cow', 'fork', 'haircut', 'waves', 'dinosaurs', 'blood', 'servant', 'structure', 'party', 'clown', 'pen', 'throat', 'frogs', 'earthq,uake', 'card', 'toothpaste', 'spy', 'worm', 'activity', 'truck', 'potato', 'office', 'island', 'bottle', 'fairies', 'maid', 'needle', 'squirrel', 'cherry', 'stone', 'advertisement', 'bells', 'kittens', 'deer', 'hobbies', 'river', 'bubble', 'soup', 'cemetery', 'coal', 'crime', 'sugar', 'thunder', 'grandfather', 'ants', 'steam', 'aftermath', 'hair', 'respect', 'wound', 'ground', 'door', 'gold', 'wish', 'fire', 'ring', 'dogs', 'theory', 'watch', 'machine', 'soda', 'star', 'locket', 'name', 'berry', 'castle', 'reward', 'pollution', 'talk', 'school', 'reaction', 'celery', 'canvas', 'car', 'airplane', 'pet', 'cakes', 'grass', 'breath', 'hand', 'snake', 'dolphin', 'home', 'directory', 'bed', 'desktop', 'partner', 'glas', 'earth', 'flower', 'rose', 'tree', 'maze', 'piano', 'ball', 'music', 'food', 'shower', 'window', 'umbrella', 'fork', 'plant', 'towel'];
```

Comment:
@andreasgruenh Thanks for the PR. Please fix the typo(glas), so I can merge it.

Reply:
Should be fixed now!